Sorting/SelectionSort/Selection sort.c

```diff
@@ -0,0 +1,45 @@
+#include<stdio.h>
+#include<conio.h>
+int smallest(int arr[],int k,int n);
+void selection_sort(int arr[],int n);
+void main()
+{
+  int arr[10],i,n;
+  clrscr();
```

Comment:
Hey, @SURYACODER is it possible for you to re-write or tweak the algorithm without using clrscr() as it is not available in most compilers these days, we need to make the repo accessible everyone.

Reply:
I have added the required changes. Kindly check and verify.